src/lib/serviceRoutinesV2/postEntities.cpp

```diff
@@ -78,10 +83,19 @@ std::string postEntities
     ciP->httpHeaderValue.push_back(location);
     ciP->httpStatusCode = SccCreated;
   }
+  else if (rcode == SccInvalidModification)
+  {
+    OrionError oe;
+    oe.code = SccContextElementNotFound;
+    oe.reasonPhrase ="AlredyExists";
+    oe.details = "Entity alredy exists";
```

Comment:
This seems like API v1 (OrionError with reasonPhrase and details).
There is another class/struct (ErrorCode) that is used for API v2.

Also, both these classes have a contructor accepting parameters, something like:

    ErrorCode errorCode("InvalidModification", "Entity already exists");

[ Not sure about the text to use for InvalidModification ... ]

It is also a bit strange to see SccContextElementNotFound in oe.code and SccInvalidModification in
ciP->httpStatusCode ...

Looking closer at OrionError (which unfortunately resides in the rest library and not in orionTypes) I see that OrionError is prepared for v2 as well (ciP carries the info on apiVersion), so OrionError can be used as well.
[ Not very good to have two different classes for this ... mea culpa ]
However, if you use OrionError, never set reasonPhrase, the constructor sets it for you, depending on the code.
Use this constructor:

    OrionError oe(SccInvalidModification, "Entity alredy exists");

    // Here is the implementation of that constructor:
    OrionError::OrionError(HttpStatusCode _code, std::string& _details)
    {
      code          = _code;
      reasonPhrase  = httpStatusCodeString(code);
      details       = _details;
    }